pkg/templates/aws/edges/listener_certificate-acm_certificate.yaml

```diff
@@ -0,0 +1,11 @@
+source: aws:listener_certificate
+target: aws:acm_certificate
+operational_rules:
+  - configuration_rules:
+      - resource: '{{ .Source }}'
+        configuration:
+          field: Certificate
+          value: '{{ .Target }}'
```

Comment:
Is it weird to have this as a edge config rule instead of an operational rule?

Reply:
can move it over